client/src/components/Sleeping Arrangments/Sleeping.js

```diff
@@ -1,19 +1,88 @@
 import React, { Component } from 'react';
 import Tile from './Tile.js';
+import { timingSafeEqual } from 'crypto';
 
 class Sleeping extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state = {
+      start: 0,
+      end: 2,
+      isRightArrow: true,
+      isLeftArrow: true
```

Comment:
Are you using airbnb style linter? Noticed there is a dangling comma missing here for airbnb style.